code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -272,4 +272,7 @@
 	if(species?.respawn_type)
 		set_death_time(species.respawn_type, world.time)
 	else
-		set_death_time(CREW, world.time)
\ No newline at end of file
+		set_death_time(CREW, world.time)
+
+/mob/living/carbon/human/get_contained_external_atoms()
+	. = contents - (internal_organs|organs)
```

Comment:
`. = ..() - organs` would also work since you remove `internal_organs` up the chain.

Reply:
gotchu